tests/integration/concrete/batch/batch.t.sol

```diff
@@ -0,0 +1,165 @@
+        );
+        // It should return nothing.
+        calls[5] = abi.encodeCall(lockup.renounce, (notTransferableStreamId));
+
+        bytes[] memory results = lockup.batch(calls);
+        assertEq(results.length, 6, "batch results length");
+        assertTrue(abi.decode(results[0], (bool)), "batch results[0]");
+        assertEq(abi.decode(results[1], (uint128)), defaults.WITHDRAW_AMOUNT(), "batch results[1]");
+        assertEq(results[2], hex"", "batch results[2]");
+        assertEq(abi.decode(results[3], (uint256)), expectedNextStreamId, "batch results[3]");
+        assertEq(abi.decode(results[4], (uint256)), expectedNextStreamId, "batch results[4]");
+        assertEq(results[5], hex"", "batch results[5]");
+    }
+
+    /// @dev The batch call includes:
+    /// - Payable functions
+    /// - All create stream functions that return a value
+    function test_BatchPayable_CreateStreams() external {
+        uint256 expectedNextStreamId = lockup.nextStreamId();
+        uint256 initialEthBalance = address(lockup).balance;
+
+        bytes[] memory calls = new bytes[](6);
+        calls[0] = abi.encodeCall(
+            lockup.createWithDurationsLD, (defaults.createWithDurations(), defaults.segmentsWithDurations())
+        );
+        calls[1] = abi.encodeCall(
+            lockup.createWithDurationsLL,
+            (defaults.createWithDurations(), defaults.unlockAmounts(), defaults.durations())
+        );
+        calls[2] = abi.encodeCall(
+            lockup.createWithDurationsLT, (defaults.createWithDurations(), defaults.tranchesWithDurations())
+        );
+        calls[3] = abi.encodeCall(lockup.createWithTimestampsLD, (defaults.createWithTimestamps(), defaults.segments()));
+        calls[4] = abi.encodeCall(
+            lockup.createWithTimestampsLL,
+            (defaults.createWithTimestamps(), defaults.unlockAmounts(), defaults.CLIFF_TIME())
+        );
+        calls[5] = abi.encodeCall(lockup.createWithTimestampsLT, (defaults.createWithTimestamps(), defaults.tranches()));
+
+        // It should return the stream IDs created.
+        bytes[] memory results = lockup.batch{ value: 1 wei }(calls);
+        assertEq(results.length, 6, "batch results length");
+        assertEq(abi.decode(results[0], (uint256)), expectedNextStreamId, "batch results[0]");
+        assertEq(abi.decode(results[1], (uint256)), expectedNextStreamId + 1, "batch results[1]");
+        assertEq(abi.decode(results[2], (uint256)), expectedNextStreamId + 2, "batch results[2]");
+        assertEq(abi.decode(results[3], (uint256)), expectedNextStreamId + 3, "batch results[3]");
+        assertEq(abi.decode(results[4], (uint256)), expectedNextStreamId + 4, "batch results[4]");
+        assertEq(abi.decode(results[5], (uint256)), expectedNextStreamId + 5, "batch results[5]");
+        assertEq(address(lockup).balance, initialEthBalance + 1 wei, "batch contract balance");
+    }
+
+    /// @dev The batch call includes:
+    /// - Payable functions
```

Comment:
It should be `ETH value` because the point is to mention that `batch` contains `{ value: 1 wei }` when making the call. Similar, in other test, the batch may not contain `{ value: 1 wei }`. 

To signal that functions are all payable in the `calls` array, I have used the test name as `test_BatchPayable_`

Reply:
Alright! Feel free to push a commit to fix this.